src/Features/LanguageServer/Protocol/Handler/Completion/CompletionResolveHandler.cs

```diff
@@ -115,6 +115,7 @@ private static LSP.VSCompletionItem CloneVSCompletionItem(LSP.CompletionItem com
                 Detail = completionItem.Detail,
                 Documentation = completionItem.Documentation,
                 FilterText = completionItem.FilterText,
+                Icon = completionItem is LSP.VSCompletionItem vsCompletionItem ? vsCompletionItem.Icon : null,
```

Comment:
Did this actually end up having any end-user impact? Just curious because I wouldn't imagine Icon's would influence the resolve step at all

Reply:
AFAIK it doesn't have a user impact, but I modified the tests in this PR and having the Icon cloned properly is required for them to pass.